financials/spending_log.csv

```diff
@@ -1,4 +1,6 @@
 spend_id,date,spend_description,decision_log_id,currency,amount,exchange_rate,USD_equivalent,from_address,to_address,tx_reference,comment
 1,20190301,Coinspect audit fee,19,BTC,4.482,3882.66,17402.08,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,17Du5PQfcLq1yeV8yw512M5UnkeZvTwe7f,https://www.blockchain.com/btc/tx/022feacd15899425ee5a164f6e67a3f8701ce4fecbad8bf4532f2576f172a468,30% advance payment
 2,20190314,Donation to Ignotus Peverell for 3 months worth of work,27,GRIN,9593.296153967,2.70,25901.9,https://donation.grin-tech.org,-,1ed0e7c7-6f15-4ed6-97f2-b1c16a6af7f3,not enough grins for full 30k payment so rest to be completed when possible
-3,20190304,Discourse forum operation costs,-,USD,3600,1,3600,-,-,-,As discussed and agreed between Igno Peverell and Blockcypher 
+3,20190304,Discourse forum operation costs,-,USD,3600,1,3600,-,-,-,As discussed and agreed between Igno Peverell and Blockcypher

```

Comment:
Does anyone remember what's the duration this payment was meanth to cover?

Reply:
From the FoG page: "BlockCypher financing last 6 months and next 2 years (so 30 months) of Discourse subscription (120 USD/month)"

Reply:
For clarity, it would be great if we could keep the review to the commits of the PR in question. Any proposed changes to other parts of the log could then be discussed in separate PRs or issues.

Reply:
ack - didn't mean to request any changes here